### PR TITLE
docs: fixed broken links

### PR DESCRIPTION
**Pull request by vinckr**
fixed broken links 
(the link in line 290 is still broken, don't know how to fix, it needs to point one level up?)

## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md).
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security. vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [x] I have added or changed [the documentation](docs/docs).

### CODE REVIEW

**Comment by aeneasr** [comment]
> fixed broken links
(the link in line 290 is still broken, don't know how to fix, it needs to point one level up?)

Yes exactly! Could you do that please? :)

**Comment by aeneasr** [comment]
@vinckr can you please finish this?

**Comment by vinckr** [comment]
I dont know how I can point it up.

File: docs/versioned_docs/version-v0.3/concepts/credentials/username-email-password.mdx [hunk 1]
```diff
@@ -287,7 +287,7 @@ Assuming your traits schema is as follows:
 
 And an identity registers with the following JSON payload (more on registration
 in
-[Selfservice Registration](../self-service/flows/user-login-user-registration.md)):
+[Selfservice Registration](../self-service/flows/user-login-user-registration)):
```
Comment (jfcurran) [inline docs/versioned_docs/version-v0.3/concepts/credentials/username-email-password.mdx:290]: [Selfservice Registration](../../self-service/flows/user-login-user-registration)):

Comment (jfcurran) [inline docs/versioned_docs/version-v0.3/concepts/credentials/username-email-password.mdx:290]: [Selfservice Registration](../../self-service/flows/user-login-user-registration)):

Comment (aeneasr) [reply docs/versioned_docs/version-v0.3/concepts/credentials/username-email-password.mdx:290]: @jfcurran what do you mean here?

@vinckr : http://archive.download.redhat.com/pub/redhat/linux/7.3/emea/doc/RH-DOCS/rhl-gsg-en-7.3/s1-navigating-cd.html

```suggestion
[Selfservice Registration](../../self-service/flows/user-login-user-registration)):
```

**Pull request merged**